airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionNotificationWorkflowImpl.java

```diff
@@ -25,15 +29,19 @@ public class ConnectionNotificationWorkflowImpl implements ConnectionNotificatio
   private NotifySchemaChangeActivity notifySchemaChangeActivity;
   @TemporalActivityStub(activityOptionsBeanName = "shortActivityOptions")
   private SlackConfigActivity slackConfigActivity;
+  @TemporalActivityStub(activityOptionsBeanName = "shortActivityOptions")
+  private ConfigFetchActivity configFetchActivity;
 
   @Override
-  public boolean sendSchemaChangeNotification(UUID connectionId, boolean isBreaking) throws IOException, InterruptedException, ApiException {
+  public boolean sendSchemaChangeNotification(UUID connectionId)
+      throws IOException, InterruptedException, ApiException, ConfigNotFoundException, JsonValidationException {
+    StandardSync standardSync = configFetchActivity.getStandardSync(connectionId);
```

Comment:
```suggestion
    final StandardSync standardSync = configFetchActivity.getStandardSync(connectionId);
```

Reply:
Looks like `final` are missing in the rest of this function. You might want to double check your IDE settings.